launchdarkly/resource_feature_flag.go

```diff
@@ -6,6 +6,14 @@ import (
 	"github.com/hashicorp/terraform/helper/schema"
 )
 
+const DEFAULT_VARIATIONS_KIND = "boolean"
+const VARIATION_NAME_KEY = "name"
```

Comment:
not sure I want to get into religion here, but those constants are used only once anyway. In those cases I typically leave them inline - the code is just easier to read that way.